### DISCUSSION TITLE
Wasm Support

### DISCUSSION THREAD

**Discussion by vdga28** [GitLiveApp/firebase-kotlin-sdk#587] [Q&A]
Is there an ETA for a firestore for the Kotllin/Wasm target?

**Comment by nbransby** [comment]
No, I don't believe anyone is actively contributing to that currently, would you like to help out?

**Comment by bpappin** [comment]
Ahh, that would explain why I'm getting an error:

:wasmJsMain: Could not resolve dev.gitlive:firebase-auth:2.1.0.
Required by:
    project :data
    
    So what would be required to get this working?

**Reply by nbransby** [reply]
latest thoughts on that here https://github.com/GitLiveApp/firebase-kotlin-sdk/issues/440
